e2e/test-component/scenarios/embedding-sdk/static-question.cy.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
```

Comment:
copy the base code from `interactive-question.cy.spec.tsx`, and leave 1 success render case + 5 new cases for loading states.